contracts/utils/rewardManager/SellingCurveRewardManager.sol

```diff
@@ -61,6 +63,7 @@ contract SellingCurveRewardManager is IRewardManager {
         rewardUsdChainlinkOracles[
             Constants.SPELL_ADDRESS
         ] = 0x8c110B94C5f1d347fAcF5E1E938AB2db60E3c9a8; // https://data.chain.link/ethereum/mainnet/crypto-usd/spell-usd
+        rewardUsdChainlinkOracles[Constants.SDT_ADDRESS] = address(0);
```

Comment:
This one can be removed. Because by default a mapping returns a default value (0 is type is uint256).